guide/how-it-works/ecash/introduction.md

```diff
@@ -0,0 +1,268 @@
+---
+layout: guide
+title: Ecash
+description: An overview of bitcoin-backed ecash, protocols, and design best practices.
+nav_order: 7
+parent: How it works
+has_children: true
+permalink: /guide/how-it-works/ecash/introduction/
+redirect_from:
+ - /guide/ecash/introductions/
+ - /guide/ecash/introduction/
+main_classes: -no-top-padding
+image: https://bitcoin.design/assets/images/guide/how-it-works/ecash/ecash.jpg
```

Comment:
Thanks! I updated this with a unique image. On what part of the site is this image referenced? I would like to see a preview of it if i can.

Reply:
okay, nice. I'll make some custom images for this case then. I think something like this will be nice:
<img width="1032" alt="image" src="https://github.com/user-attachments/assets/393e469f-4373-440b-add8-76f4b42b0fa0">